recipes/pasta/meta.yaml

```diff
@@ -1,5 +1,5 @@
-{% set version = "1.9.0" %}
-{% set sha256 = "6f816aebe00b7b6bb78a250ad45f5aa2aa8eff0f9719736f05b0597dd3b1f9a1" %}
+{% set version = "1.9.2" %}
```

Comment:
I think it would be best to update title of this PR so it will be obvious from the first line of the squashed commit that pasta was also updated.

Reply:
done